torch/distributed/algorithms/ddp_comm_hooks/default_hooks.py

```diff
@@ -18,9 +18,7 @@ def allreduce_hook(
         >>> ddp_model.register_comm_hook(process_group, allreduce_hook)
     """
     group_to_use = process_group if process_group is not None else dist.group.WORLD
-    world_size = (
-        process_group.size() if process_group is not None else dist.get_world_size()
-    )
+    world_size = process_group.size()
```

Comment:
On the line above, it looks like `process_group` can be None. Shouldn't this be `world_size = group_to_use.size()`?

Reply:
Yes, obviously I copied and pasted the wrong word :(